core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -5914,15 +5925,15 @@ protected function filterMenu($intFilterPanel)
 						$option_label = $vv ?: '-';
```

Comment:
Is this correct? Previously the label would have been set to `$vv` which makes more sense to me I think.

Reply:
You have commented on a line that has not changed. Are you referring to line 5919? If yes, then I agree. 😄

Reply:
Fixed in 1550fd769ce350112719147253a37736f5c7cd79.

Reply:
I tried to comment on multiple lines, but apparently I failed ☺️

> Are you referring to line 5919? If yes, then I agree. 😄

Yes.